.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
For the code review, but it's fine, GitHub allows you to see the changes as individual commits or all combined. When the PR is ready to be merged, though, I think that having a single commit is cleaner.